pkg/exec/executor_backoff_test.go

```diff
@@ -27,8 +30,8 @@ func (s *ExecutorBackoffTestSuite) SetupTest() {
 			Blocking:        false,
 			CancelDelay:     0,
 			InitialInterval: time.Millisecond,
-			MaxInterval:     time.Second,
-			MaxElapsedTime:  time.Second,
+			MaxInterval:     time.Millisecond * 3,
+			MaxElapsedTime:  time.Millisecond * 10,
```

Comment:
This seems to be not enough for some tests when one has bad luck with scheduling. Maybe this needs to be decided on a test by test basis?

Reply:
gave it a little more space